app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -494,6 +494,7 @@ class EventDetailsFragment : Fragment() {
             onEventClick = eventClickListener
             onFavFabClick = favFabClickListener
         }
+
```

Comment:
Extra file committed, reset looks good to me.

Reply:
@anhanh11001

Reply:
fixed